File: app/src/main/java/com/example/sportivo/Court.java
package com.example.sportivo;

import com.example.sportivo.objects_screen.Company;

public class Court {
    private int courtId;
    private String courtName;
    private int sportId;
    private int companyId;
    private Company company;
    private int price;

    public Court(String name, int sportId, int companyId, int price){
        this.courtName = name;
        this.sportId = sportId;
        this.companyId = companyId;
        this.price = price;
    }

    public Court(){

    }

    public int getCourtId() { return courtId; }
    public String getCourtName(){ return courtName; }
    public Company getCompany() { return company; }
    public int getSportId() { return sportId ;}
    public int getPrice() { return price; }
}

File: Sportivo_webAPI/Models/Court.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Sportivo_webAPI.Models
{
    public class Court
    {
        public int CourtId { get; set; }
        public string CourtName { get; set; }
        public Company Company { get; set; }
        public int CompanyId { get; set; }
        public Sport Sport { get; set; }
        public int SportId { get; set; }
        public int Price { get; set; }
        public ICollection<Reservation> Reservations { get; set; }
    }
}
